prep-css-syntax/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>CSS Syntax</title>
+</head>
+<body>
+  <h1 style="font-family: Arial, Helvetica, sans-serif;">This h1 tag has been given "Arial" as a font-family.</h1>
+  <p style="font-size: 2rem;">I am a p tag which has a font-size of 2rem!!!</p>
+  <h3 style="color: firebrick;">This h3 tag has a font color of firebrick.</h3>
+  <h2 style="font-style: italic;">This h2 has an "italic" font-style.</h2>
```

Comment:
Hi JC, for this exercise you should put your styles inside of some `<style></style>` tags in the document's `<head>`

Reply:
Thanks Tim for letting me know my mistake I went ahead and fixed it but I accidentally merged this one 😓